src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/VsProjectAdapter.cs

```diff
@@ -35,18 +35,18 @@ internal class VsProjectAdapter : IVsProjectAdapter
 
         #region Properties
 
-        public string BaseIntermediateOutputPath
+        public string MSBuildProjectExtensionsPath
         {
             get
             {
-                var baseIntermediateOutputPath = BuildProperties.GetPropertyValue(ProjectBuildProperties.BaseIntermediateOutputPath);
+                var msbuildProjectExtensionsPath = BuildProperties.GetPropertyValue(ProjectBuildProperties.MSBuildProjectExtensionsPath);
```

Comment:
We need to make sure that this works in VS. 
We need to make sure this property is correctly evaluated and returned by the ProjectSystem.

Reply:
I have verified a project.json project from command line and VS and it works perfectly fine.